### PR TITLE
Add more information to process_suspect_commits stats

### PR DESCRIPTION
**Pull request by wedamija**
We want to be able to determine whether a specific project is causing issues with suspect commits,
so adding more detail here.

### CODE REVIEW

**Comment by wedamija** [comment]
I can add transactions here as well if we think it's helpful.

**Review by untitaker — COMMENTED** [review]
not sure about adding tag to statsd metric, we can stomach extra transactions on group creation more easily afaik

File: src/sentry/tasks/groupowner.py
Comment (tonyo) [inline src/sentry/tasks/groupowner.py:26]: Let's not add project IDs as tags to `metrics.*` calls, please. It will open doors to high-cardinality metrics that make our metrics provider not very happy.

**Comment by untitaker** [comment]
@wedamija took the liberty to push this forward, I believe this should work now

**Review by wedamija — COMMENTED** [review]
@untitaker thanks, this has been in my todo pile. I'll merge once approved

**Review by untitaker — APPROVED** [review]

File: src/sentry/utils/sdk.py
Comment (tonyo) [inline src/sentry/utils/sdk.py:67]: Would this mean enabling tracing also for onpremise?

**Review by tonyo — APPROVED** [review]

File: src/sentry/utils/sdk.py
Comment (untitaker) [reply src/sentry/utils/sdk.py]: fixed to use a setting

**Pull request merged**
